www/html/ci/application/controllers/Landing.php

```diff
@@ -3,15 +3,28 @@
 
 class Landing extends CI_Controller {
 
+	public $loader;
+	public $twig;
```

Comment:
@If there is no reason for these to be public, then we should make them protected/private. This follows Object oriented designs